jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerClientTest.java

```diff
@@ -267,4 +265,25 @@ public void testTag_fail() throws InterruptedException, InvalidImageReferenceExc
       Assert.assertEquals("'docker tag' command failed with error: error", ex.getMessage());
     }
   }
+
+  private DockerClient makeDockerSaveClient() {
+    return new DockerClient(
+        subcommand -> {
+          try {
+            if (subcommand.contains("{{.Size}}")) {
+              Process mockSize = Mockito.mock(Process.class);
+              Mockito.when(mockSize.getInputStream())
+                  .thenReturn(new ByteArrayInputStream("150000".getBytes(StandardCharsets.UTF_8)));
```

Comment:
Is the number 150000 used somewhere? Or can it be just any number? The above assertion says ` Assert.assertEquals(3, counter[0])`, so I'm confused. Any why is it 3? Does it write exactly 3 bytes? If so, worth commenting why it is 3.

Reply:
The 3 is because "jib" is what the `InputStream` actually writes, so yes it writes exactly 3 bytes. The 150000 is just a placeholder number. I'll add comments.